test/sqllogictest/windowed-aggregates.slt

```diff
@@ -0,0 +1,79 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# window_tumbling
+
+query T
+SELECT window_tumbling(ts, '1m')
+FROM (
+    VALUES ('2001-01-01 01:02:34'::timestamp)
+) vals (ts);
+----
+("2001-01-01 01:02:00","2001-01-01 01:03:00")
```

Comment:
I like the record-based approach here, I just wonder if it won't cause any performance/optimizer issues?

Reply:
Can you elaborate on your concern here?

Reply:
Solved via Slack, the issues I was thinking of (predicate propagation) aren't valid here anyway.